backend/src/bridge.rs

```diff
@@ -0,0 +1,390 @@
+use axum::Json;
+use dotenvy::dotenv;
+use jsonrpsee::core::client::ClientT;
+use jsonrpsee::core::ClientError;
+use log::{info, error, warn};
+use jsonrpsee::http_client::HttpClient;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use std::{env, sync::Arc};
+use strata_bridge_primitives::types::PublickeyTable;
+use strata_bridge_rpc::types::{
+    RpcOperatorStatus,
+    RpcDepositInfo,
+    RpcDepositStatus,
+    RpcWithdrawalInfo,
+    RpcWithdrawalStatus,
+    RpcClaimInfo,
+    RpcReimbursementStatus,
+};
+use tokio::{sync::Mutex, time::{Duration, interval}};
+
+use crate::utils::create_rpc_client;
+
+pub struct BridgeMonitoringConfig {
+    strata_rpc_url: String,
+    bridge_rpc_url: String,
+    stats_refetch_interval_s: u64,
+}
+
+impl BridgeMonitoringConfig {
+    pub fn new() -> Self {
+        dotenv().ok(); // Load `.env` file if present
+
+        let strata_rpc_url = env::var("STRATA_RPC_URL").ok()
+            .unwrap_or_else(|| {
+                "http://127.0.0.1:8545".to_string()
+            });
+
+        let bridge_rpc_url = env::var("STRATA_BRIDGE_RPC_URL").ok()
+            .unwrap_or_else(|| "http://127.0.0.1:8546".to_string());
+
+        let refresh_interval_s = env::var("BRIDGE_STATUS_REFETCH_INTERVAL_S").ok()
+            .unwrap_or_else(|| "120000".to_string());
+        let refetch_interval_s_u64: u64 = refresh_interval_s.parse().expect("Failed to parse BRIDGE_STATUS_REFETCH_INTERVAL_S as u64");
+
+        info!("🔹 Strata rpc url {}, bridge rpc url {}", strata_rpc_url, bridge_rpc_url);
```

Comment:
The way we log this is not using the `{}`. We generally don't do that in `alpenlabs` repos. All the `trace::` macros you can (and should) do:

```suggestion
        info!(%strata_rpc_url, %bridge_rpc_url, "some important context without emojis");
```
And you can use `%` for `fmt::Display` and `?` for `fmt::Debug` traits.

Reply:
I was using log::info, not trace. I will make the switch.